Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -61,8 +65,19 @@ CompositingCoordinator::CompositingCoordinator(WebPage& page, CompositingCoordin
     , m_client(client)
 #if USE(CAIRO)
     , m_paintingEngine(Nicosia::PaintingEngine::create())
+#elif USE(SKIA)
+    , m_skiaAcceleratedBufferPool(nullptr)
+    , m_skiaUnacceleratedThreadedRenderingPool(nullptr)
```

Comment:
This is not needed, std::unique_ptr already initializes its pointer nullptr

Reply:
D'oh.